src/drivers/cmakefileapi/api_helpers.ts

```diff
@@ -49,6 +48,13 @@ export async function loadCacheContent(filename: string): Promise<Map<string, ap
   const file_content = await fs.readFile(filename);
   const cache_from_cmake = JSON.parse(file_content.toString()) as index_api.Cache.CacheContent;
 
+  const expected_version = {major: 2, minor: 0};
+  const detected_version = cache_from_cmake.version;
+  if (detected_version.major != expected_version.major || detected_version.minor != expected_version.minor) {
+    log.warning(`Cache object version (${detected_version.major}.${detected_version.minor}) of FileAPI is not as ` +
```

Comment:
All warnings should be localized.  Can you call the `localize` function for this and extract out the variables?
```
localize(
  'cache.object.version',
  'Cache object version ({0}.{1}) of cmake-file-api is unexpected. Expecting ({2}.{3}). IntelliSense configuration may be incorrect.',
  detected_version.major,
  detected_version.minor,
  expected_version.major,
  expected_version.minor)
```
```